packages/react-ui-core/src/Gmap/GmapInteraction.js

```diff
@@ -0,0 +1,20 @@
+import once from 'lodash/once'
+
+class GmapCallbackFactory {
+  constructor() {
+    let map = {}
+    /* eslint-disable consistent-return */
+    this.registerMap = once(newMap => {
+      map = newMap
+    })
+
+    this.call = (f, args) => {
+      if (map[f] && typeof map[f] === 'function') {
+        return map[f](...args)
+      }
+    }
+    /* eslint-enable consistent-return */
```

Comment:
Is it possible to return null or undefined here to make the linter happy, if nothing else?

Reply:
Done!